account_asset_analytic/models/account_asset_depreciation_line.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+
+
+class AccountAssetDepreciationLine(models.Model):
+    _inherit = 'account.asset.depreciation.line'
+
+    @api.multi
+    def create_move(self):
+        move_ids = super(AccountAssetDepreciationLine, self).create_move()
+        for line in self:
+            asset = line.asset_id
+            analytic_id = asset.analytic_account_id.id
+            expense_id = asset.category_id.account_expense_depreciation_id.id
+            if line.move_id and analytic_id:
+                for move_line in line.move_id.line_id:
```

Comment:
`for line in line.move_id.line_id.filtered(lambda x: x.account_id == expense_account)`